### ISSUE TITLE
[New Client]: type issue when deleting files.

### ISSUE THREAD

**Issue by shyakadavis** [david-plugge/typed-pocketbase#33]
Hi;

Don't know how I missed this (was not using `.from()`) but basically, when you're updating a field to delete an image, for example, you need to:
> set the file field to a zero-value
(null, [], empty string, etc.).

source: https://pocketbase.io/docs/files-handling/#deleting-files

Currently, files are typed as `File | undefined`, that means if I do
```ts
await event.locals.pb.from('projects').update(event.params.project_id, { thumbnail: null });
```

I get: `Type 'null' is not assignable to type 'File | undefined'.ts(2322)`, and setting that field to `undefined` does not delete it from PB.

May you please update the types to account for any zero-value type. In the past I was using `''`, or `null` to delete a file.

Thanks.

**Comment by david-plugge** [comment]
Hey, good catch. Thanks for reporting, i'll include the fix in the next release

**Comment by david-plugge** [comment]
Should be fixed in `0.1.0`!

**Comment by shyakadavis** [comment]
Works splendidly. Thank you. 🙂
